joomla/joomla-cms
# Clean up some Jlayouts

Pull Request for Issue # .
#### JLayouts should not be used to inject javascript!
### Summary of Changes

Inline scripts moved to their respective files and utilise the Joomla Options!
### Testing Instructions

Apply patch and check that:
- Chosen still works
- Tags still works
- drag and drop in a list view (e.g. articles) still works
### B/C

Totally backwards compatible
### Documentation Changes Required

None
